resources/views/layouts/navbar.blade.php

```diff
@@ -12,6 +12,22 @@
     </div>
 </a>
 
+@if(env('APP_SHOW_BRANCH'))
```

Comment:
I think that it's perhaps marginally better to use the `config` helper where we can, rather than `env` helper.  In order to slightly decouple configuration from the .env file.  We'd still use the .env file for now, but makes it easier to move should we ever wish for settings to come from elsewhere, e.g. the database.

That said, certainly not a big deal here, and would be good to get your opinion on this - let's discuss at the retro.